packages/react/src/transactions/adapters/useUnfollowController.ts

```diff
@@ -48,7 +48,7 @@ export function useUnfollowController() {
 
     const unfollowProfile = new UnfollowProfile(signedFollow, gateway, transactionQueue, presenter);
 
-    await unfollowProfile.execute(request);
+    void unfollowProfile.execute(request);
```

Comment:
Something I couldn't figure out, but with the presenter pattern implemented by `TransactionResultPresenter` we need to await or early errors won't show up.

Reply:
I changed that because the UI feels slow but I will revert if that doesn't work as expected